app/components/report-row/component.js

```diff
@@ -27,8 +29,10 @@ export default class ReportRowComponent extends Component {
    * @public
    */
   @action
-  save(changeset) {
-    this.args.onSave(changeset);
+  async save(changeset) {
```

Comment:
use `dropTask` for that and you won't need any tracking variable. Further add `this.save.isRunning` to the `disabled` state check in the template.